app/frontend/src/features/profile/view/References.tsx

```diff
@@ -47,6 +77,19 @@ export default function References() {
         <Typography className={classes.header} variant="h1">
           {REFERENCES}
         </Typography>
+        {availableReferences &&
+          availableReferences.canWriteFriendReference &&
+          user.friends === User.FriendshipStatus.FRIENDS && (
+            <div className={classes.buttonContainer}>
+              <Link
+                to={{
+                  pathname: `${leaveReferenceBaseRoute}/friend/${user.userId}`,
```

Comment:
If this is hardcoded to always go to a friend reference... how do we access writing a host/surfer reference?